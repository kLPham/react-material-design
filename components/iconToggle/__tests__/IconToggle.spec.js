import React from 'react';
import { shallow } from 'enzyme';
import { IconToggle } from '../index';

describe('<IconToggle />', () => {
    it('should be able to render', () => {
        const icon = 'favorite_border';
        const wrapper = shallow(
            <IconToggle icon={icon} />,
  );
        expect(wrapper).toMatchSnapshot();
    });
});
