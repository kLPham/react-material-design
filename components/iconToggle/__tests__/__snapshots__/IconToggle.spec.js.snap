exports[`<IconToggle /> should be able to render 1`] = `
<i
  aria-disabled={false}
  className="mdc-icon-toggle material-icons"
  onClick={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  role="button"
  tabIndex="0">
  favorite_border
</i>
`;
